whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -548,6 +554,7 @@ class PostgreSQLComponent {
                 Document dependerDoc = assembleDocument(resultSet)
                 if (linkFinder != null)
                     linkFinder.normalizeIdentifiers(dependerDoc, connection)
+                dependerDoc.setModified(modTime)
```

Comment:
This gets a little backwards. If a record A has been merged with record B, it does not follow that record C (which happens to link to A) should have its modified-time updated.

Reply:
OK.
But the old code updates `lddb.modified` for `dependerDoc`. Should we change that behavior?

Reply:
Ahh, you're right again of course! Well done :)